db/migrations/20220721152441_increase_size_of_scroll_id_column.js

```diff
@@ -0,0 +1,15 @@
+exports.up = function up(knex) {
+  const result = knex.schema
+    .alterTable('work_items', (t) => {
+      t.string('scrollID', 4096).alter();
+    });
+
+  return result;
+};
+
+exports.down = function down(knex) {
+  return knex.schema
```

Comment:
I think in this case I wouldn't allow a rollback to reduce the column size since it will likely break if there are any IDs created greater than 32 characters. I would just put a comment explaining why the rollback does nothing.

Reply:
Done.